mesonbuild/options.py

```diff
@@ -103,6 +103,8 @@ class ArgparseKWs(TypedDict, total=False):
 }
 
 _BAD_VALUE = 'Qwert Zuiopü'
+_OptionKey__cache: T.Dict[T.Tuple[str, str, MachineChoice], OptionKey] = {}
```

Comment:
Can we just do `_optionkey_cache`?

Reply:
yes. At first, I though I could directly use `__cache` from within the class, but it doesn't work like that...